include/zephyr/zbus/zbus.h

```diff
@@ -0,0 +1,507 @@
+/*
+ * Copyright (c) 2022 Rodrigo Peixoto <rodrigopex@gmail.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_ZBUS_H_
+#define ZEPHYR_INCLUDE_ZBUS_H_
+
+#include <string.h>
+
+#include <zephyr/kernel.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Zbus API
+ * @defgroup zbus_apis Zbus APIs
+ * @{
+ */
+
+/**
+ * @brief Type used to represent a channel.
+ *
+ * Every channel has a zbus_channel structure associated used to control the channel
+ * access and usage.
+ */
+struct zbus_channel {
+	/** Channel name. */
+	const char *name;
```

Comment:
Shouldn't this field be optional (in `zbus_observer` as well)? I guess it's only for debugging/logging purposes.